src/Microsoft.Framework.PackageManager/Commands/WrapCommand.cs

```diff
@@ -692,7 +692,7 @@ private static string GetNuGetPackagePath(string packagesDir, string id, string
 
         private static string ResolvePackagesDirectory(string projectDir)
         {
-            var rootDir = ProjectResolver.ResolveRootDirectory(projectDir);
```

Comment:
It is used to locate the root of current solution (the ancestor folder containing `global.json`). This method is used widely in our runtime and functional tests.